### PR TITLE
修改了英文API文档

### PR DESCRIPTION
**Pull request by Atlantisming**
### PR types
Others

### PR changes
Docs

### Describe
修改的API文档有：
- paddle.nn.dynamic_decode
- paddle.nn.functional.diag_embed
- paddle.set_grad_enabled
- paddle.einsum
- paddle.unique_consecutive
- paddle.disable_signal_handler
- paddle.linalg.svd
- paddle.cumprod
- paddle.device.cuda.stream_guard
- paddle.incubate.segment_min
- paddle.incubate.segment_max
- paddle.incubate.segment_sum
- paddle.incubate.segment_mean

对应的中文修复文档为：[PaddlePaddle/docs/#5429](https://github.com/PaddlePaddle/docs/pull/5429)

### CODE REVIEW

File: python/paddle/incubate/tensor/math.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def segment_sum(data, segment_ids, name=None):
                             For more information, please refer to :ref:`api_guide_Name`.
 
     Returns:
-       output (Tensor): the reduced result.
+       Tensor: the reduced result.
```
Comment (SigureMo) [inline python/paddle/incubate/tensor/math.py]: 这里应该是 `,`，`:` 的话会解析成两行

下同

File: python/paddle/incubate/tensor/math.py [hunk 2]
```diff
@@ -28,7 +28,7 @@
     reason="paddle.incubate.segment_sum will be removed in future",
 )
 def segment_sum(data, segment_ids, name=None):
-    r"""
+    """
```
Comment (SigureMo) [inline python/paddle/incubate/tensor/math.py]: 带公式的 docstring 是建议加 `r` 的，这样可以保证中英文文档里的公式是一致的，不需要在英文里写双倍的 `\` 来转义，可参考 https://github.com/PaddlePaddle/Paddle/pull/46752#issuecomment-1275551496

所以，只需要移除下面公式里重复的 `\` 即可，如 `$out_i = \\sum_{j} data_{j}$` -> `$out_i = \sum_{j} data_{j}$`

不带公式的且无需转义的可以考虑移除 `r`

下同

File: python/paddle/incubate/tensor/math.py (hunk 1 above)
Comment (Atlantisming) [reply python/paddle/incubate/tensor/math.py]: 已修改

File: python/paddle/incubate/tensor/math.py (hunk 2 above)
Comment (Atlantisming) [reply python/paddle/incubate/tensor/math.py]: 已修改

Comment (Atlantisming) [reply python/paddle/incubate/tensor/math.py]: 像官方书写文档[https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/dev_guides/api_contributing_guides/api_docs_guidelines_cn.html#id6](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/dev_guides/api_contributing_guides/api_docs_guidelines_cn.html#id6)
里面英文模板一样的公式写法
.. math::
           out = x+y
这种格式的文档是不是也移除 `r`

Comment (SigureMo) [reply python/paddle/incubate/tensor/math.py]: 唔，是说多行公式？多行公式和行内公式一样，其实问题不是带不带公式，而是带不带转义字符 `\`，只不过大多数带公式的 docstring 一般都会包含 `\`。

> 这种格式的文档是不是也移除 `r`

为啥要移除呀 :joy:，带公式（或者说带转义）应该加上呀，移除里面重复的 `\`

Comment (Atlantisming) [reply python/paddle/incubate/tensor/math.py]: 噢噢好的，所以官方文档里的公式因为不包含转义符号所以就没有定义成原始字符串。
我不太清楚英文文档生成的语法规则。

File: python/paddle/nn/functional/extension.py [hunk 3]
```diff
@@ -57,48 +57,48 @@ def diag_embed(input, offset=0, dim1=-2, dim2=-1):
     Examples:
```
Comment (Ligoml) [inline python/paddle/nn/functional/extension.py:61]: api描述的地方把 This OP 这类错误描述去掉吧

File: python/paddle/fluid/layers/rnn.py [hunk 4]
```diff
@@ -1824,7 +1824,6 @@ def dynamic_decode(
```
Comment (Ligoml) [inline python/paddle/fluid/layers/rnn.py:1822]: 返回值直接罗列吧，说返回一个tuple好像没啥意义，可以参考这个：[paddle.geometric.reindex_graph](https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/geometric/reindex_graph_en.html)

File: python/paddle/tensor/manipulation.py [hunk 5]
```diff
@@ -2296,7 +2296,7 @@ def unique_consecutive(
             :ref:`api_guide_Name`. Default is None.
 
     Returns:
-        tuple (out, inverse, counts). `out` is the unique consecutive tensor for `x`. `inverse` is provided only if `return_inverse` is True. `counts` is provided only if `return_counts` is True.
+        tuple(out, inverse, counts): `out` is the unique consecutive tensor for `x`. `inverse` is provided only if `return_inverse` is True. `counts` is provided only if `return_counts` is True.
```
Comment (Ligoml) [inline python/paddle/tensor/manipulation.py]: 同 paddle.nn.dynamic_decode 修改建议

Comment (Ligoml) [reply python/paddle/tensor/manipulation.py]: note中的 paddle.unique 需要增加一个超链接，方法参考：[飞桨文档相互引用](https://github.com/PaddlePaddle/docs/wiki/%E9%A3%9E%E6%A1%A8%E6%96%87%E6%A1%A3%E7%9B%B8%E4%BA%92%E5%BC%95%E7%94%A8)

File: python/paddle/tensor/linalg.py [hunk 6]
```diff
@@ -2047,7 +2047,7 @@ def svd(x, full_matrices=False, name=None):
             For more information, please refer to :ref:`api_guide_Name`.
```
Comment (Ligoml) [inline python/paddle/tensor/linalg.py:2040]: full_matrices 有默认值，是可选参数，需要注明optional，并说明默认值是什么

Comment (Ligoml) [reply python/paddle/tensor/linalg.py:2040]: 中文文档格式有点问题，需要是 ` - **full_matrices** (bool，可选) - 参数描述` 这种格式

Comment (Ligoml) [reply python/paddle/tensor/linalg.py:2040]: tuple改一下吧，中文部分似乎已经改了

File: python/paddle/tensor/math.py [hunk 7]
```diff
@@ -3437,7 +3437,7 @@ def cumprod(x, dim=None, dtype=None, name=None):
         name (str, optional): Name for the operation (optional, default is None). For more information, please refer to :ref:`api_guide_Name`.
 
     Returns:
-        Tensor, the result of cumprod operator.
+        Tensor: the result of cumprod operator.
```
Comment (Ligoml) [inline python/paddle/tensor/math.py]: 用逗号吧，这个的问题在于 dim 是一个可选参数，中英文文档都需要改

File: python/paddle/device/cuda/__init__.py [hunk 8]
```diff
@@ -355,8 +355,8 @@ def _set_current_stream(stream):
 @signature_safe_contextmanager
 def stream_guard(stream):
     '''
-    **Notes**:
-        **This API only supports dygraph mode currently.**
+    Notes:
+        This API only supports dygraph mode currently.
```
Comment (Ligoml) [inline python/paddle/device/cuda/__init__.py]: 英文文档改对了，中文文档也需要补充哦

File: python/paddle/incubate/tensor/math.py [hunk 9]
```diff
@@ -163,7 +163,7 @@ def segment_min(data, segment_ids, name=None):
 
     This operator calculate the minimum elements of input `data` which with
     the same index in `segment_ids`.
-    It computes a tensor such that $out_i = \\min_{j} data_{j}$
+    It computes a tensor such that $out_i = \min_{j\in \{segment\_ids_j==i\}} data_{j}$
```
Comment (Ligoml) [inline python/paddle/incubate/tensor/math.py]: image

公式显示奇奇怪怪的，能不能像中文那样公式单独一行居中？

Comment (Ligoml) [reply python/paddle/incubate/tensor/math.py]: 另外中文部分，张量直接写作 Tensor 即可，翻译过来反而增加理解上的困难

**Comment by Ligoml** [comment]
顺便说一下，你现在可以在commit名称后面加上 `; test=docs_preview` 来触发文档预览的CI，这样就可以根据预览效果调整啦~参考这个 https://github.com/PaddlePaddle/Paddle/pull/48281

File: python/paddle/tensor/manipulation.py (hunk 5 above)
Comment (Ligoml) [reply python/paddle/tensor/manipulation.py]: 似乎还没改完？

File: python/paddle/tensor/linalg.py [hunk 10]
```diff
@@ -2038,16 +2038,21 @@ def svd(x, full_matrices=False, name=None):
...
 
     Returns:
-        Tuple of 3 tensors: (U, S, VH). VH is the conjugate transpose of V. S is the singlar value vectors of matrics with shape `[..., K]`
+        - U(Tensor), is the singular value decomposition result U.
+
+        - S(Tensor), is the singular value decomposition result S.
+
+        - VH(Tensor), VH is the conjugate transpose of V, which is the singular value decomposition result V.
+        Tuple of 3 tensors(U, S, VH): VH is the conjugate transpose of V. S is the singlar value vectors of matrics with shape `[..., K]`
```
Comment (Ligoml) [inline python/paddle/tensor/linalg.py:2047]: image

报了个错，看上去需要在 Tuple of 3 tensors 之前增加一个空行

File: python/paddle/device/cuda/__init__.py (hunk 8 above)
Comment (Ligoml) [inline python/paddle/device/cuda/__init__.py]:
```suggestion
        This API only supports dynamic graph mode currently.
```

File: python/paddle/incubate/tensor/math.py (hunk 2 above)
Comment (Atlantisming) [reply python/paddle/incubate/tensor/math.py]: 似乎行内公式在没有移除`r`的情况下，只有一个转义字符`\`的话会导致这样的结果
> image
> 
查询了英文文档的生成方式之后，找不到合适的公式表达
> 公式显示奇奇怪怪的，能不能像中文那样公式单独一行居中？

猜测是带`r`的注释，在不转义的情况下是直接将整段字符串传给sphinx，再进行一次编译？
sphinx中的`\`仍为一个转义符号？

Comment (SigureMo) [reply python/paddle/incubate/tensor/math.py]: 直接从中文里 copy 过来即可，我已经修改完成

**Comment by SigureMo** [comment]
英文里的问题基本都改了下，不过没看预览不保证都对，中文那边就不帮忙改了

**Comment by Ligoml** [comment]
image

中文文档这里多了个括号，需要改一下 @Atlantisming

**Review by Ligoml — APPROVED** [review]
good job！LGTM for docs

**Review by luotao1 — APPROVED** [review]
